File: sample/tests/java/com/example/java/ComplicatedAssertionTest.java
package com.example.java;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNull;
import static org.junit.Assert.assertTrue;

import org.junit.Test;

public class ComplicatedAssertionTest {
    @Test
    public void test() {
        assertEquals("Hello", "World");

        assertTrue(false);

        String s = "Hello";
        assertNull(null, s);
    }
}

File: sample/tests/kotlin/com/example/kotlin/ComplicatedAssertionTest.kt
package com.example.kotlin

import kotlin.test.Test
import kotlin.test.assertEquals
import kotlin.test.assertNull
import kotlin.test.assertTrue

class ComplicatedAssertionTest {
    @Test
    fun test() {
        assertEquals("Hello", "World")

        assertTrue(false)

        val s: String? = "Hello"
        assertNull(null, s)
    }
}
